robot-core/src/main/java/org/obolibrary/robot/DiffOperation.java

```diff
@@ -112,35 +112,58 @@ public static boolean compare(
       return false;
     }
 
-    writer.write(strings1minus2.size() + " axioms in Ontology 1 but not in Ontology 2:\n");
-    sorted = new TreeSet<>();
-    for (String axiom : strings1minus2) {
-      if (useLabels) {
-        sorted.add("- " + addLabels(ioHelper, labels, axiom) + "\n");
-      } else {
-        sorted.add("- " + axiom + "\n");
-      }
-    }
-    for (String axiom : sorted) {
-      writer.write(axiom);
+    OWLOntologySetProvider ontologyProvider =
+        new DualOntologySetProvider(
+            ontology1.getOWLOntologyManager(), ontology2.getOWLOntologyManager());
+
+    switch (format.toLowerCase()) {
+      case "simple":
+        if (useLabels) {
+          DefaultPrefixManager pm = ioHelper.getPrefixManager();
+          AnnotationValueShortFormProvider labelProvider =
+              new AnnotationValueShortFormProvider(
+                  ontologyProvider,
+                  pm,
+                  pm,
+                  Collections.singletonList(OWLManager.getOWLDataFactory().getRDFSLabel()),
+                  Collections.emptyMap());
+          OBOShortenerShortFormProvider iriProvider = new OBOShortenerShortFormProvider(pm);
+          DoubleShortFormProvider doubleProvider =
+              new DoubleShortFormProvider(iriProvider, labelProvider);
+          writer.write(BasicDiffRenderer.render(diff, doubleProvider));
+        } else {
```

Comment:
With simple format, no labels, I get outputs like:
`- SubClassOf(<http://purl.obolibrary.org/obo/ECO_0000000> owl:Thing)`

When using labels, I see:
`- SubClassOf(<ECO:0000000>[evidence] <owl:Thing>)`

Could you use `OBOShortenerShortFormProvider iriProvider` for the else case of `useLabels` as well?

Reply:
I think so—you're wanting it to look like this, correct?

`- SubClassOf(ECO:0000000 owl:Thing)`

Reply:
Addressed by c0597da90e1cb5360eeb439324f0e6dcc2bb768f.